dds/DCPS/Serializer.cpp

```diff
@@ -364,11 +364,12 @@ Serializer::~Serializer()
 {
 }
 
-Serializer::ScopedAlignmentContext::ScopedAlignmentContext(Serializer& ser)
+Serializer::ScopedAlignmentContext::ScopedAlignmentContext(Serializer& ser, size_t min_read)
```

Comment:
What about a unit test where min_read is tested?

Reply:
Added. I didn't realize I needed min_read until I was having issues parsing our version parameter, which says it's 4 bytes but only reads two octets. So I didn't have a unit test for it at first, but good call, it should be there.